### DISCUSSION TITLE
My account has been flagged !?

### DISCUSSION THREAD

**Discussion by HersheyPlus** [community/community#74546] [New to GitHub]
My account has been flagged and I can't pay github copilot

**Comment by Balmasexy** [comment]
What help do you need as an assistant

**Comment by Balmasexy** [comment]
WhatsApp me let's talk on WhatsApp

**Comment by Akash1134** [comment]
Hi @HersheyPlus ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Our private support team is the only one able to help with support requests on account flags. You will need to follow up with them directly through your open ticket. Thanks!

**Comment by lamrelia** [comment]
Did you register or use a throwaway or spam email. Because if you didn't do any legitimately wrong that may be the culprit
